src/Inkluzitron/Services/ImagesService.cs

```diff
@@ -21,73 +25,156 @@ namespace Inkluzitron.Services
 {
     public class ImagesService
     {
-        private ProfilePictureService ProfilePictureService { get; }
+        private DiscordSocketClient Client { get; }
+        private ImageCacheSettings CacheSettings { get; }
 
-        public ImagesService(ProfilePictureService profilePictureService)
+        private DirectoryInfo AvatarCache { get; }
```

Comment:
To vypadá divně. Co takhle ještě jednu classu, která tu bude jako property s názvem Cache a v ní budou properties pak pojmenované jako Avatar, Whip, Love, ... ?

Reply:
Přidal jsem DI-singleton třídu FileCache a upravil ukázku použití (viz popis PR), aby šly tyhle individuální properties zahodit.